src/monaco-adapter.ts

```diff
@@ -62,10 +62,10 @@ export class MonacoAdapter implements IEditorAdapter {
     this._remoteCursors = new Map<ClientIDType, IRemoteCursor>();
     this._cursorWidgetController = new CursorWidgetController(this._monaco);
 
-    this._redo = null;
-    this._undo = null;
-    this._originalRedo = null;
-    this._originalUndo = null;
+    this._redo = undefined;
```

Comment:
I am afraid I don't understand the difference!

Reply:
typescript was crying because of mismatched types and won't let the build happen.